docs/user-guide/admin-cli.md

```diff
@@ -39,17 +39,17 @@ optional arguments:
 ### Create
 A tenant named "_DEFAULT" will be created automatically post install.
 
-Creates a new named tenant and optionally assigns VMs.
+Creates a new named tenant and optionally assigns VMs. Valid tenant name is only allowed to be "[a-zA-Z0-9_][a-zA-Z0-9_.-]*"
```

Comment:
It would be helpful if you can add some useful information in the PR title as well as in the first comment. 

Such information will be useful for future reference while searching from title.  Current title proposing the ```Fixes issue 816 (space in DS names).```